<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>import_multiple_wells_Dialog</class>
 <widget class="QDialog" name="import_multiple_wells_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multiple Wells</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Well name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X (m)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y (m)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>KB (m)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>WD (m)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TD (m)</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open File</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>import_multiple_wells_Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
